docs/cli/gittuf_clone.md

```diff
@@ -9,8 +9,9 @@ gittuf clone [flags]
 ### Options
 
 ```
-  -b, --branch string   specify branch to check out
-  -h, --help            help for clone
+  -b, --branch string      specify branch to check out
+  -h, --help               help for clone
+  -k, --root-key strings   path(s) of root keys to verify when cloning the repo.
```

Comment:
`-k` seems like a misleading shorthand, because in the rest of gittuf `-k` is used for a private key.

Reply:
also a good point!

Reply:
Fixed